delta-lake/delta-spark350db143/src/main/scala/com/nvidia/spark/rapids/delta/DeltaProbe.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, NVIDIA CORPORATION.
+ * Copyright (c) 2025, NVIDIA CORPORATION.
```

Comment:
Isn't this a pre-existing file?
```suggestion
 * Copyright (c) 2024-2025, NVIDIA CORPORATION.
```

Reply:
Yeah. I don't know what happened there.